src/Components/MovieCard/index.js

```diff
@@ -14,25 +14,29 @@ export default function MovieCard(props) {
   const handleClose = () => setShow(false);
   const handleShow = () => setShow(true);
 
+  const showMovieDetails = useCallback(() => props.showMovieDetails(props.movie.id), [props.showMovieDetails, props.movie.id]);
```

Comment:
I'd recommend to distructure props, so you don't need to write `props` every time

it's just a nice practice, makes your lines a bit cleaner. Check other components too, I believe there are many places where you can substructure props like that